testsuite/features/support/minion_checks.rb

```diff
@@ -1,7 +1,7 @@
 # check if the minion is a SLE-15
 def minion_is_sle15
   node = get_target('sle-minion')
-  os_version = get_os_version(node, false)
-  return false if os_version.nil?
-  os_version.include? '15'
+  os_version, os_family = get_os_version(node)
+  return false if os_version.nil? || os_family.nil?
+  (os_version.include? '15') && (os_family.include? 'sles')
```

Comment:
In fact, to be correct we should use always regex, as this would match `16.15` :-)

But, we can leave with this.

Reply:
I thought at it and 1) `=~ /15/` is same as `include? "15"`and 2) we don't have 16.15 :smile_cat: 

But yeah, I'll fix. Stay tuned.